commerce_coordinator/apps/stripe/pipeline.py

```diff
@@ -189,4 +189,10 @@ def run_filter(
             logger.info(f'[CT-{tag}] Unsuccessful Stripe refund with details:'
                         f'[order_id: {order_id}, payment_intent_id: {payment_intent_id}'
                         f'message_id: {kwargs["message_id"]}')
+
+            if 'greater than unrefunded amount on charge' in ex.user_message:
```

Comment:
we didn't have any other param except this message to identify this. `ex.code` is empty, `ex.param` is `amount`, which is not reliable.